modules/data-loading/pages/data-streaming-connector/aws-s3.adoc

```diff
@@ -33,23 +33,40 @@ For connecting to AWS S3, its value is
 `com.tigergraph.kafka.connect.filesystem.aws.S3SourceConnector`.
 
 ==== Provide AWS credentials
-The connector uses the standard simple credential provider and uses your https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html[access key] for authentication.
+The connector can use two methods for authentication.
 
-The following fields are required for authentication:
+* The standard simple credentials provider and your https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html[access key].
+* AWS Identity and Access Management (IAM) roles.
+
+
+The configuration fields are as follows:
 
 * `file.reader.settings.fs.s3a.aws.credentials.provider`: Comma-separated class names of credential provider classes which implement
 `com.amazonaws.auth.AWSCredentialsProvider`.
 ** This field should be set to `org.apache.hadoop.fs.s3a.SimpleAWSCredentialsProvider`.
```

Comment:
I think this line also needs update mentioning users can use this provider if they'd like to provide access key and secret key.